juniper_hyper/Cargo.toml

```diff
@@ -13,11 +13,11 @@ futures = "0.3.1"
 juniper = { version = "0.15.3", path = "../juniper", default-features = false }
 hyper = "0.13"
 serde_json = "1.0"
-tokio = "0.2"
+tokio = "1.0.2"
```

Comment:
Did this get reverted? It seems like tokio is back to 0.2 on the latest master as of writing: https://github.com/graphql-rust/juniper/blob/master/juniper_hyper/Cargo.toml#L16.

Reply:
nvm just realized this was closed in favor of #888